File: Homeworks/1. Programming/1. C#-Part-1/Other-Solutions/03.Operators-and-Expressions/05. Third Digit/ThirdDigit.cs
/*Problem 5. Third Digit is 7?
Write an expression that checks for given integer if its third digit from right-to-left is 7.
*/
namespace _05.Third_Digit
{
    using System;

    class ThirdDigit
    {
        static void Main()
        {
            int numberToCheck = int.Parse(Console.ReadLine());

            IsThirdDigit7(numberToCheck);
        }

        static void IsThirdDigit7(int numberToCheck)
        { 
            int thirdDigit = (numberToCheck / 100) % 10;
            bool isThirdDigit7 = thirdDigit == 7;

            // Console.WriteLine(isThirdDigit7 ? "true" : $"false {thirdDigit}");
            Console.WriteLine(isThirdDigit7 ? "true" : "false " + thirdDigit);
        }
    }
}

File: Homeworks/Web Design and UI technlogies/3. JavaScript-Part-1/02.Operators and Expressions/04. Third digit/ThirdDigit.js
/**Problem 4. Third digit
 Write an expression that checks for given integer if its third digit (right-to-left) is 7.
 */

function CheckThirdDigit(number) {
    var digit = (number / 100) % 10 | 0,
        check = digit == 7; // round with | 0

    if (check) {
        return console.log(check);
    } else {
        return console.log('false ' + digit);
    }
}

CheckThirdDigit(877);